GenerativeIT. test failing

### CI Link

https://gradle-enterprise.elastic.co/s/3l2b7f3juduia

### Repro line

./gradlew ":x-pack:plugin:esql:qa:server:multi-node:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.multi_node.GenerativeIT.test" -Dtests.seed=A3FAD75259885065 -Dtests.locale=es-UY -Dtests.timezone=America/Coral_Harbour -Druntime.java=25

### Does it reproduce?

Didn't try

### Applicable branches

main

### Failure history

_No response_

### Failure excerpt

Seems like its still failing periodically.

```
GenerativeIT > test FAILED
--
java.lang.AssertionError: query: SET unmapped_fields="nullify";from airport_city_boundaries \| stats  city = values(abbrev) \| inline stats  XTJaAutI = first(city, null) by city
exception: method [POST], host [http://[::1]:37395], URI [/_query?pretty=true&error_trace=true], status line [HTTP/1.1 400 Bad Request]
Warnings: [No limit defined, adding default limit of [1000]]
{
"error" : {
"root_cause" : [
{
"type" : "illegal_argument_exception",
"reason" : "Found a single entry with 100001 entries",
"stack_trace" : "org.elasticsearch.ElasticsearchException$1: Found a single entry with 100001 entries\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.ElasticsearchException.guessRootCauses(ElasticsearchException.java:808)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.ElasticsearchException.generateFailureXContent(ElasticsearchException.java:722)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.rest.RestResponse.build(RestResponse.java:202)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:161)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.rest.RestResponse.<init>(RestResponse.java:122)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.rest.action.RestActionListener.onFailure(RestActionListener.java:54)\n\tat org.elasticsearch.xpack.esql.action.EsqlResponseListener.lambda$wrapWithLogging$1(EsqlResponseListener.java:191)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:78)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.action.ActionListener$2.onFailure(ActionListener.java:282)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.rest.action.RestCancellableNodeClient$1.onFailure(RestCancellableNodeClient.java:97)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:78)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:89)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.action.ActionListener$3.onFailure(ActionListener.java:442)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.tasks.TaskManager$1.onFailure(TaskManager.java:235)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:78)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:89)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.action.DelegatingActionListener.onFailure(DelegatingActionListener.java:32)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations$RunBeforeActionListener.onFailure(ActionListenerImplementations.java:360)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeAcceptException(ActionListenerImplementations.java:78)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.action.ActionListenerImplementations.safeOnFailure(ActionListenerImplementations.java:89)\n\tat org.elasticsearch.server@9.4.0-SNAPSHOT/org.elasticsearch.action.ActionListener$3.onFailure(ActionListener.java:442)\n\tat org.elasticsearch.server@9.4.0-


```

--- a/x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/aggregation/blockhash/PackedValuesBlockHash.java
+++ b/x-pack/plugin/esql/compute/src/main/java/org/elasticsearch/compute/aggregation/blockhash/PackedValuesBlockHash.java
@@ -265,7 +265,7 @@ final class PackedValuesBlockHash extends BlockHash {
                         ords.appendInt(Math.toIntExact(found));
                         count++;
                         if (count > Block.MAX_LOOKUP) {
-                            // TODO replace this with a warning and break
+                            // TODO replace this with a warning and break and remove the accepted error from GenerativeRestTest
                             throw new IllegalArgumentException("Found a single entry with " + count + " entries");
                         }
                     }

--- a/x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/generative/GenerativeRestTest.java
+++ b/x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/generative/GenerativeRestTest.java
@@ -85,6 +85,11 @@ public abstract class GenerativeRestTest extends ESRestTestCase implements Query
         "failed to parse date field \\[.*\\] with format",
         // full-text function trying to parse a non-IP string
         "is not an IP string literal",
+        // a values(<that field>) agg could more than 100,000 values into a single multi-valued field, and a subsequent
+        // inline stats … by <that field> hits the hard limit Block.MAX_LOOKUP = 100_000 in the compute layer
+        // throwing IllegalArgumentException via PackedValuesBlockHash
+        // see https://github.com/elastic/elasticsearch/issues/145694
+        "Found a single entry with .* entries",
 
         // Awaiting fixes for query failure
         "Unknown column \\[<all-fields-projected>\\]", // https://github.com/elastic/elasticsearch/issues/121741,

